### PR TITLE
feat(sdk): added company id and name in activity log list

### PR DESCRIPTION
**Pull request by xuelianhan007**
## Description

Added support of showing company id and name in activity log list.
## Related Issue

https://github.com/mycloudnexus/kraken/issues/33

## How This Has Been Tested
Integration test and verified in internal development environment.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
- [ ] CI/CD or documentation update (changes to CI/CD pipeline or documentation)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Review by kuangxiang20240501 — DISMISSED** [review]
LGTM

File: kraken-java-sdk/kraken-java-sdk-controller/src/test/java/com/consoleconnect/kraken/operator/controller/BuyerCreator.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import com.consoleconnect.kraken.operator.core.model.HttpResponse;
+import com.consoleconnect.kraken.operator.core.toolkit.JsonToolkit;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+
+public interface BuyerCreator {
+  String PRODUCT_ID = "product.mef.sonata.api";
+  String BUYER_BASE_URL = String.format("/products/%s/buyers", PRODUCT_ID);
+  String BUYER_ID = "consolecore-poping-company";
```
Comment (zhiyaoPCCW) [inline kraken-java-sdk/kraken-java-sdk-controller/src/test/java/com/consoleconnect/kraken/operator/controller/BuyerCreator.java:18]: Replace the name with a mock one.

**Review by zhiyaoPCCW — CHANGES_REQUESTED** [review]

File: kraken-java-sdk/kraken-java-sdk-controller/src/test/java/com/consoleconnect/kraken/operator/controller/BuyerCreator.java (hunk 1 above)
Comment (xuelianhan007) [reply kraken-java-sdk/kraken-java-sdk-controller/src/test/java/com/consoleconnect/kraken/operator/controller/BuyerCreator.java]: Thanks. The test data  has been renamed.

**Review by kuangxiang20240501 — APPROVED** [review]

**Review by zhiyaoPCCW — APPROVED** [review]
LGTM

**Pull request merged**